### PR TITLE
fix: linting

### PR DESCRIPTION
**Pull request by levino**
still working on this

### CODE REVIEW

File: apps/homepage2/.eslintrc.js [hunk 1]
```diff
@@ -1,21 +1,11 @@
...
     '@next/next/no-html-link-for-pages': 'off',
   },
-  settings: {
-    react: {
-      version: 'detect',
-    },
+  env: {
+    browser: true,
+    es2021: true,
+    jquery: true,
```
Comment (levino) [inline apps/homepage2/.eslintrc.js:9]: This enables the usage of `$` in the code.

File: apps/homepage2/package.json [hunk 2]
```diff
@@ -52,13 +53,11 @@
     "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.24.5",
```
Comment (levino) [inline apps/homepage2/package.json:55]: I am not sure that I removed all unnecessary dependencies. Where did you use this dependency?

File: packages/jsonresume-theme-flat/package.json [hunk 3]
```diff
@@ -12,13 +12,7 @@
...
   "dependencies": {
-    "handlebars": "latest",
-    "prismjs": "^1.29.0"
+    "handlebars": "latest"
   },
-  "license": "MIT",
-  "devDependencies": {
-    "@babel/eslint-parser": "^7.24.5",
-    "eslint": "^9.3.0",
-    "eslint-plugin-react": "^7.33.2"
```
Comment (levino) [inline packages/jsonresume-theme-flat/package.json:22]: I think in general, these are unnecessary. Removed them.

File: packages/jsonresume-theme-flat/package.json [hunk 4]
```diff
@@ -12,13 +12,7 @@
...
   },
   "dependencies": {
-    "handlebars": "latest",
-    "prismjs": "^1.29.0"
+    "handlebars": "latest"
   },
-  "license": "MIT",
-  "devDependencies": {
-    "@babel/eslint-parser": "^7.24.5",
-    "eslint": "^9.3.0",
```
Comment (levino) [inline packages/jsonresume-theme-flat/package.json:21]: Unfortunately this is quite dangerous. We use `eslint@8`. `eslint@9` brings breaking changes with regards to the [configuration](https://eslint.org/docs/latest/use/configure/migration-guide#custom-parsers) and bringing it into the project will create all kinds of nightmares.

File: eslint.config.js
Comment (levino) [inline eslint.config.js:1]: @thomasdavis this was the main mistake, I think. Adding this file means ["opting into the new flat config"](https://eslint.org/docs/v8.x/use/configure/configuration-files-new) for `eslint`. `eslint` will then ignore all `.eslintrc` files.

**Review by levino — COMMENTED** [review]
This should fix any issues we encountered.

Important lessons:
- do not use `eslint@9` or above (we can do it, but we also need to migrate all configs).
- do not add `eslint.config.js` files because they enable the new ["flat config"](https://eslint.org/docs/v8.x/use/configure/configuration-files-new) mode of `eslint@8`.

File: apps/homepage2/package.json (hunk 2 above)
Comment (thomasdavis) [reply apps/homepage2/package.json:55]: can't remember +1 to remove

File: eslint.config.js
Comment (thomasdavis) [reply eslint.config.js:1]: yeah great to know

**Review by thomasdavis — APPROVED** [review]
This is awesome, great work. 

- noted to not go to eslint9 (i'm in no rush)

**Pull request merged**